api/sixtyfps-rs/lib.rs

```diff
@@ -185,7 +185,9 @@ struct MyStruct {
 
 pub use sixtyfps_macros::sixtyfps;
 
-pub use sixtyfps_corelib::graphics::{Brush, Color, Image, LoadImageError, RgbaColor};
+pub use sixtyfps_corelib::graphics::{
+    Brush, Color, Image, LoadImageError, RGB8Pixel, RGBA8Pixel, RgbaColor, SharedPixelBuffer,
```

Comment:
Now we have both RGBA8Pixel and RgbaColor in the public API

Note the difference in the uppercasing of rgba.
Also do we still need RgbaColor?

Reply:
I think we should keep the `Pixel` types, in contrast to the `Color` types.

The casing is a good point though, that is an inconsistency. I'll change it to `Rgb8Pixel` and `Rgba8Pixel`.